src/coreclr/utilcode/stresslog.cpp

```diff
@@ -154,15 +154,19 @@ void StressLog::Initialize(unsigned facilities, unsigned level, unsigned maxByte
 
     theLog.lock = ClrCreateCriticalSection(CrstStressLog, (CrstFlags)(CRST_UNSAFE_ANYMODE | CRST_DEBUGGER_THREAD | CRST_TAKEN_DURING_SHUTDOWN));
     // StressLog::Terminate is going to free memory.
+    size_t maxBytesPerThread = maxBytesPerThreadArg;
```

Comment:
Do we want to be limited by pointer size? Perhaps using `ULONGLONG` instead?

Reply:
Actually, pointer size is a natural limit because all of the stresslog is stored in memory. So I think it's fine as is.